packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -664,54 +715,57 @@ Slider.propTypes = {
    * Should not be equal to max.
    */
   min: PropTypes.number,
+  /**
+   * Name attribute of the hidden `input` element.
+   */
+  name: PropTypes.string,
+  /**
+   * Callback function that is fired when the mouseup is triggerd.
+   *
+   * @param {object} event The event source of the callback
+   * @param {any} value The new value
+   */
+  onAfterChange: PropTypes.func,
```

Comment:
onAfterChange implies it is always called after onChange. Would prefer onChangeCommitted which is closer to when onChange should actually be fired.

Reply:
Ok, good for me, I'm also triggering it in for the keyboard events then.

Reply:
Which is how onInput/onChange would work as well.